packages/api-react/src/services/wallet.ts

```diff
@@ -175,17 +175,17 @@ export const walletApi = apiWithTag.injectEndpoints({
       walletId: number;
       poolUrl: string;
       relativeLockHeight: number;
-      targetPuzzlehash?: string;
+      targetPuzzleHash?: string;
       fee?: string;
     }>({
-      query: ({ walletId, poolUrl, relativeLockHeight, targetPuzzlehash, fee }) => ({
```

Comment:
The `pw_join_pool` RPC looks like it expects the parameter name to be `target_puzzlehash`.  `pwJoinPool()` in Wallet.ts seems correct. Is this name change correct if the RPC expects `target_puzzlehash` as the param name?

Reply:
Pool services return target_puzzle_hash which is converted to targetPuzzleHash but chia API is using target_puzzlehash which is targetPuzzlehash. I renamed it because it was not working

Reply:
Ahh ok, so pwJoinPool still uses `target_puzzlehash`, which is populated by the value in `targetPuzzleHash`. Sounds good.

Reply:
yes :)